recipes/_packaging.rb

```diff
@@ -58,3 +62,5 @@
   omnibus_env['PATH'] << node['wix']['home']
   omnibus_env['PATH'] << node['7-zip']['home']
 end
+
+include_recipe 'omnibus::_libffi' if ppc64? || ppc64le?
```

Comment:
If I recall - it's because the libffi libraries aren't found by default on these platforms, and as such ruby/gem things from later down the chain will fail building the extension. @kaustubh-d can confirm?